docs/iris/src/whatsnew/contributions_v2.0.0/incompatiblechange_2017-Oct-18_remove-analysis-interpolation.txt

```diff
@@ -0,0 +1,8 @@
+* The deprecated module ```iris.analysis.interpolate``` has been removed, along with the following functions:
+    * ```iris.analysis.linear```
+    * ```iris.analysis.all_points_in_range```
```

Comment:
This is a method in `Linear1DExtrapolator`, which technically was never deprecated (outside of being part of the deprecated class), so it should not be in this list.